src/queueDisplay/index.ts

```diff
@@ -0,0 +1,177 @@
+import { EasingFunction, Entity, GltfContainer, Material, MeshRenderer, PBVisibilityComponent, PlayerIdentityData, TextAlignMode, TextShape, Transform, TransformType, Tween, VisibilityComponent, engine } from "@dcl/sdk/ecs";
+import { Color4, Quaternion, Vector3 } from "@dcl/sdk/math";
+import { getPlayer } from "@dcl/sdk/src/players";
+import * as utils from "@dcl-sdk/utils"
+import * as playersQueue from "../queue/index"
+import { getSDK } from "../sdk";
+
+export enum SCREENS {
+    addToQueue,
+    playNext,
+    queueList
+}
+
+let frameEntity: Entity
+let positionActive: TransformType
+let positionDisabled: TransformType
+let displayEntity: Entity
+let waitingListEntity: Entity
+let myPosEntity: Entity
+let active = false
+let currentScreen: number
+let screensAtlas = 'assets/scene/GameSigns.png'
```

Comment:
this is not going to work. move the asset inside the `mini-game-models` folder so it can be copied inside the local scene